app/templates/compare_revisions.html

```diff
@@ -63,27 +63,19 @@
     <div class="grid-row">
       <div class="column-two-thirds page-section">
         <p>
-          {% if all_update_events is none %}
-            Multiple
+          {% if latest_update_events %}
+            {% if number_of_users_who_made_edits > 1 %}
+              The last user to edit this page was
```

Comment:
Not sure about the term "this page". Should try and keep it to the idea they're editing a *service* not just a "page".

Reply:
Happy to change 'page' to 'service'.

Reply:
Cool, I've changed that now :).